scalding-core/src/main/scala/com/twitter/scalding/serialization/CascadingBinaryComparator.scala

```diff
@@ -53,6 +54,7 @@ object CascadingBinaryComparator {
 
     def check(s: Splice): Try[Unit] = {
       val m = s.getKeySelectors.asScala
+      val sortingSelectors = s.getSortingSelectors.asScala
```

Comment:
Happy to just drop it entirely, is it worth adding a new option for? I'm not sure it is, secondary sorting is rare enough we can have users manually specify it if in doubt?